### PR TITLE
Styling changes

### PR DESCRIPTION
**Pull request by WheresHJ**
General styling for navbar, facets, and infobox. Needs mobile and further detail-level typography on facets and infobox.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -25,7 +25,8 @@
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
-         <a class="navbar-brand" data-project="name" href="">Finda</a>
+         <img class="brand_image" src="img/finda_small.png" /> <!-- This is the logo of the deploying organization -->
```
Comment (paulswartz) [inline index.html]: - if you give this a `data-project=="logo"` attribute it'll be override-able in the config
- alt text?

**Comment by ohnorobo** [comment]
I like the purple :+1:

**Comment by ohnorobo** [comment]
The facet scrollbar goes past the bottom of the page for me.

File: index.html (hunk 1 above)
Comment (WheresHJ) [inline index.html]: Good adds, I'll put them in. Alt text should reflect the deploying organization as well.

**Comment by WheresHJ** [comment]
@ohnorobo Whoops, had a fix for that in inspector, didn't commit. Thanks for the catch. The purple is just a CfA accent color from http://style.codeforamerica.org

**Comment by WheresHJ** [comment]
Ok, should be set. As I said, some detail typography left to do on the facets and the infobox, then I'll move on to mobile.

**Pull request merged**